apps/condo/domains/property/gql.js

```diff
@@ -26,6 +26,9 @@ const PROPERTY_MAP_GRAPHQL_TYPES = `
     enum BuildingUnitType {
         parking
         flat
+        apartment
```

Comment:
These values should be imported from consts

Reply:
Fixed this